sources/agent/nerdydata/nerdydata.go

```
@@ -0,0 +1,131 @@
+			var err error
+			for attempt := 0; attempt < maxRetries; attempt++ {
+				resp, err = agent.queryURL(session, nerdydataRequest.buildURL(), session.Keys.NerdyDataToken)
+				if err != nil {
+					results <- sources.Result{Source: agent.Name(), Error: err}
+					return
+				}
+				if resp.StatusCode != http.StatusAccepted {
+					break
+				}
+				// 202 = server-side timeout; back off and retry with same cursor
+				resp.Body.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)